test/GMapAssoc.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Was this committed by accident? I don't see how it's related to the patch.

Reply:
^^"......accident...I jump to the wrong test folder....I will change it next time.